### DISCUSSION TITLE
WebGIS6 - Drucken

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#189] [Q&A]
Hallo Jürgen!

Ich teste gerade das Drucken im WebGIS6 (6.24.2803). Das Anlegen einer Subpage für die Legende (wie im Workshop gezeigt) funktioniert super. Aber gibt es zum Drucken irgendwo eine Doku? Welche Tags gibt es bei der Erstellung der Layouts, welche Parameter kann ich verwenden? Ich finde dazu leider nichts. 

Im Vergleich zum WebGIS5 (5.24.1101) ist mir beim Drucken ein Unterschied aufgefallen. Bei uns ist in der Kopfzeile das Niederösterreich-Logo eingebunden. Im **WebGIS5** schaut die Kopfzeile je nach Druckqualität (**96 oder 150 dpi**) immer gleich aus --> 
![image](https://github.com/user-attachments/assets/41ed4016-54b7-4ac4-a08a-517db8ce9f0e)

Im **WebGIS6** kommt es auf die Druckauflösung an. Je nachdem welche Druckqualität der User einstellt ändert sich die Größe des Logos:
**96 dpi -->**
![image](https://github.com/user-attachments/assets/2ec2a15e-ce42-4da8-bc17-257b3a946fd4)

**150 dpi -->** 
![image](https://github.com/user-attachments/assets/4d3e15c1-2d5f-41c2-a9a6-0a6ad6648d5b)

Die Kopfzeile ist in der Druck.xml folgendermaßen konfiguriert
`<panel dock="top" fillcolor="0,90,154" height="7"  id="TopFrameNoeAtlasSchriftzug">
			<image src="img/logos/watermark_noe_logo.gif" width="7" x="0" y="1" height="6" />
			<text string="IMAP &#160;&#160; [TITLE]" x="8" y="2" font="Verdana" fontstyle="bold" fontsize="2,5" fontcolor="255,213,0" />
		</panel>`

Wir haben die Druckqualität in der api.config festgelegt
`<add key="qualities-dpi" value="96:Standard,120:Mittel,150:Hoch" />`

- Muss ich beim WebGIS6 in der Konfig eine Änderung vornehmen, um die Größe des Logos in der Kopfzeile bei unterschiedlichen Druckqualitäten beizubehalten?
- Wie könnte ich verhindern, dass das Logo bei unterschiedlichen Druckqualitäten die Größe ändert?
- Gibt es eine Doku für die Erstellung von Drucklayouts im WebGIS?

Danke und einen schönen Montag!

Lg, Andi

**Comment by gis-eni** [comment]
Eine Beschreibung der neuen Parameter gibt es derzeit nur in den Folien des Workshops 2024.

Den Bug mit den Logos habe ich mit Build 6.24.3101 behoben.

**Comment by Andreas-Lampert** [comment]
Vielen Dank! Funktioniert!
